tests/framework/common.go

```diff
@@ -1173,6 +1170,25 @@ func (td *OsmTestData) CreateDockerRegistrySecret(ns string) {
 	}
 }
 
+// ShouldCleanupK8sOnKind returns whether k8s cleanup can be avoided on kind when the
+// kind cluster is not going to be kept anyway, when the test/suite is running on kind.
+// This optimization considerably speeds up the test suite when running on kind as we don't wait
+// for k8s cleanup after every test.
+func (td *OsmTestData) ShouldCleanupK8sOnKind(ct CleanupType) bool {
```

Comment:
un-export this function: s/ShouldCleanupK8sOnKind/shouldCleanupK8sOnKind

Reply:
will do